<?xml version="1.0"?>
<launch>
<rosparam command="load" file="$(find ur5_intpro)/config/ur5_params.yaml" />

<arg name="ur5_ip" default="192.168.1.3" />
<arg name="ur5_port" default="30003" />
<arg name="r2fg_port" default="/dev/ttyUSB0"/>

<param name="ur5/ip" value="$(arg ur5_ip)" />
<param name="ur5/port" value="$(arg ur5_port)" />
<param name="r2fg/port" value="$(arg r2fg_port)" />

<node name="ur5" type="ur5" pkg="irl_robots" output="screen"/>
<node name="r2fg" type="r2fg" pkg="irl_robots" output="screen"/>
</launch>
